docs/faq/frequently_asked_questions.rst

```diff
@@ -0,0 +1,56 @@
+
+.. include:: ../text_colors.rst
+.. toctree::
+
+.. _frequently_asked_questions:
+
+##########################
+Frequently Asked Questions
+##########################
+
+#. I just purchased my motor, how do I get started with Control Center? How can I communicate with my module?
```

Comment:
I think if someone comes to this page before doing anything (which...if they're here, there's a good chance of), seeing "Control Center" would be a bit confusing. What's, "Control Center?" 

Might be worth specifying a bit like, "how do I get started with Vertiq's IQ Control Center software"